project/AppDependencies.scala

```diff
@@ -22,28 +22,28 @@ object AppDependencies {
 
   val compile: Seq[ModuleID] = Seq(
     ws,
-    "uk.gov.hmrc"   %% "bootstrap-play-26" % "1.5.0",
-    "uk.gov.hmrc"   %% "domain"            % "5.9.0-play-26",
-    "uk.gov.hmrc"   %% "play-hmrc-api"     % "4.1.0-play-26",
-    "org.typelevel" %% "cats-core"         % "2.2.0",
-    compilerPlugin("com.github.ghik" % "silencer-plugin" % "1.6.0" cross CrossVersion.full),
-    "com.github.ghik" % "silencer-lib" % "1.6.0" % Provided cross CrossVersion.full,
+    "uk.gov.hmrc"   %% "bootstrap-backend-play-27" % "3.4.0",
+    "uk.gov.hmrc"   %% "domain"            % "5.10.0-play-27",
+    "uk.gov.hmrc"   %% "play-hmrc-api"     % "5.3.0-play-27",
+    "org.typelevel" %% "cats-core"         % "2.4.0",
+    compilerPlugin("com.github.ghik" % "silencer-plugin" % "1.7.0" cross CrossVersion.full),
+    "com.github.ghik" % "silencer-lib" % "1.7.0" % Provided cross CrossVersion.full,
     "com.chuusai"   %% "shapeless"         % "2.4.0-M1"
   )
 
   def test(scope: String = "test, it"): Seq[sbt.ModuleID] = Seq(
-    "org.scalatest"          %% "scalatest"          % "3.2.0"             % scope,
-    "com.vladsch.flexmark"   % "flexmark-all"        % "0.35.10"           % scope,
+    "org.scalatest"          %% "scalatest"          % "3.2.3"             % scope,
+    "com.vladsch.flexmark"   % "flexmark-all"        % "0.36.8"           % scope,
     "org.scalacheck"         %% "scalacheck"         % "1.14.2"            % scope,
-    "org.scalamock"          %% "scalamock"          % "5.0.0"             % scope,
+    "org.scalamock"          %% "scalamock"          % "5.1.0"             % scope,
     "com.typesafe.play"      %% "play-test"          % PlayVersion.current % scope,
-    "org.scalatestplus.play" %% "scalatestplus-play" % "3.1.2"             % scope,
+    "org.scalatestplus.play" %% "scalatestplus-play" % "4.0.3"             % scope,
     "com.github.tomakehurst" % "wiremock"            % "2.27.2"            % scope
   )
 
   // Fixes a transitive dependency clash between wiremock and scalatestplus-play
   val overrides: Seq[ModuleID] = {
```

Comment:
Are these necessary? AFAIK the flexmark upgrade should make these redundant - though I could be wrong

Reply:
Are you referring to all 4 dependencies there? or a specific one?